src/components/about-chat-sidebar/AboutChatSidebar.tsx

```diff
@@ -0,0 +1,144 @@
+import { FC, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { useTranslation } from 'react-i18next'
+
+import Typography from '@mui/material/Typography'
+import Box from '@mui/material/Box'
+import Divider from '@mui/material/Divider'
+import Avatar from '@mui/material/Avatar'
+
+import IconButton from '@mui/material/IconButton'
+import ImageOutlinedIcon from '@mui/icons-material/ImageOutlined'
+import InsertDriveFileOutlinedIcon from '@mui/icons-material/InsertDriveFileOutlined'
+import LinkOutlinedIcon from '@mui/icons-material/LinkOutlined'
+import Close from '@mui/icons-material/Close'
+
+import AppButton from '~/components/app-button/AppButton'
+import SidebarContentBox from '~/components/sidebar-content-box/SidebarContentBox'
+import SidebarImageGrid from '~/components/sidebar-image-grid/SidebarImageGrid'
+import FileComponent from '~/components/file-component/FileComponent'
+import LinkComponent from '~/components/link-component/LinkComponent'
+
+import { SizeEnum, ButtonVariantEnum, UserResponse, Link, File } from '~/types'
+import { createUrlPath, spliceSx } from '~/utils/helper-functions'
+import { authRoutes } from '~/router/constants/authRoutes'
+
+import { styles } from '~/components/about-chat-sidebar/AboutChatSidebar.styles'
+
+interface AboutChatSidebarProps {
+  user: UserResponse
+  media: Array<string>
+  files: File[]
+  links: Link[]
+}
+
+const AboutChatSidebar: FC<AboutChatSidebarProps> = ({
```

Comment:
Also, if this component will only be used for chat, better to move it from components to containers

Reply:
> Also, if this component will only be used for chat, better to move it from components to containers

Yeah that make sense. moved 👍